src/config.go

```diff
@@ -29,6 +29,7 @@ type Config struct {
 	Blocks               []*Block          `config:"blocks"`
 	Tooltips             []*Segment        `config:"tooltips"`
 	TransientPrompt      *TransientPrompt  `config:"transient_prompt"`
+	Palette              Palette           `config:"palette"`
```

Comment:
this needs a pointer

Reply:
I thought about it, and... Does it really need a pointer? I'm just not sure. I looked this question up during development and found a few sources that state it is not required. As far as I understand, map types are reference types, they are already pointers. https://stackoverflow.com/questions/28384343/accessing-a-map-using-its-reference, https://go.dev/blog/maps. The same as Blocks, for example. It is []*Block, not *[]*Block, event though it can be. But still,  If a pointer is a requirement of the Config module or just a required convention, I can do that.

Reply:
You are correct, I mistook it for a struct. Should've looked better.